### PR TITLE
Us036 deploy api

### PR DESCRIPTION
**Pull request by joshuasy10**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      # Runs a set of commands using the runners shell
+      - name: Run a multi-line script
+        run: |
+          echo Add other actions to build,
+          echo test, and deploy your project.
```
Comment (JamesMcKee-Kainos) [inline .github/workflows/deploy.yml:32]: This bit doesn't look like it does anything, can it just be removed?

Comment (joshuasy10) [reply .github/workflows/deploy.yml:32]: removed

**Review by JamesMcKee-Kainos — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+name: "deploy"
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```
Comment (ccurran01) [inline .github/workflows/deploy.yml:11]: Do we want to deploy every change that is pushed?

**Review by ccurran01 — COMMENTED** [review]
Have you been able to successfully use the API endpoints?

**Review by ccurran01 — APPROVED** [review]

**Pull request merged**